scripts/globals/spells/black/retrace.lua

```diff
@@ -2,28 +2,16 @@
 -- Spell: Retrace
 -- Transports player to their Allied Nation. Can cast on allies.
 -----------------------------------
-require("scripts/globals/teleports")
-require("scripts/globals/status")
-require("scripts/globals/msg")
+require("scripts/globals/spells/spell_enhancing_teleport")
```

Comment:
If there's multiple enhancing scripts in the global, perhaps we should move this into a sub-directory?  Just thinking about really long names.